src/components/tx/modals/BatchExecuteModal/ReviewBatchExecute.tsx

```diff
@@ -91,6 +112,19 @@ const ReviewBatchExecute = ({ data, onSubmit }: { data: BatchExecuteData; onSubm
         </Typography>
         <DecodedTxs txs={txsWithDetails} />
 
+        {canRelay ? (
+          <>
+            <Typography mt={2} color="primary.light">
+              How to pay:
+            </Typography>
+            <SponsoredBy
```

Comment:
Can we add a slight margin bottom to "How to pay:" like with the other caption?
<img width="569" alt="Screenshot 2023-03-30 at 12 31 40" src="https://user-images.githubusercontent.com/5880855/228809402-27146df7-5ec3-4e9a-b865-f9274bd86b0a.png">